app/models/condition.rb

```diff
@@ -0,0 +1,10 @@
+class Condition < ApplicationRecord
+  validates_presence_of :date,
```

Comment:
A thought - do you think all these validations will cause us trouble later? I haven't dug through the data, but I would guess that some fields may be missing in some places. We can test this out with the fixture data tomorrow.

Reply:
I think we should ensure all data is present, whether that requires normalizing some data in the seeds or  inputs from a user. I think we’d rather have some value than nil so that our pages don’t through errors.